conn_process.go

```diff
@@ -32,6 +32,8 @@ type onProcess struct {
 	progress      func(*Progress)
 	profileInfo   func(*ProfileInfo)
 	profileEvents func([]ProfileEvent)
+	gotData       func()
```

Comment:
What's the difference of `gotData` vs `data`? Other than the function signature?